vcs-diff-lint-csdiff-ruff

```diff
@@ -0,0 +1,64 @@
+#! /usr/bin/python3
+
+"""
+The csdiff tool doesn't support the Ruff's JSON output yet.  So this just a
+trivial wrapper which reads Ruff's report and transforms it to JSON which is
+supported by csdiff.
+
+The script accepts the same parameters as `ruff check` itself.
+"""
+
+import os
+import sys
+import json
+from subprocess import Popen, PIPE
+
+
+def ruff_check():
+    """
+    Run `ruff check` and return a dict with its results
+    """
+    cmd = ["ruff", "check", "--output-format=json"] + sys.argv[1:]
```

Comment:
Should it be added as a weak deps to spec file?

Should the argv be sanitized?

Reply:
> Should it be added as a weak deps to spec file?

It should, thank you. Good catch.

> Should the argv be sanitized?

I don't know, I did the same that `vcs-diff-lint-csdiff-pylint` does. But probably not? Worst case scenario, the bad values are going to be passed to `ruff check` which will handle it and error out.

Reply:
Right, with the shell=False (that is the default) this is not a problem.